plugins/funind/recdef.ml

```diff
@@ -141,7 +141,7 @@ let def_id = Id.of_string "def"
 let p_id = Id.of_string "p"
 let rec_res_id = Id.of_string "rec_res";;
 let lt = function () -> (coq_init_constant "lt")
-let le = function () -> (coq_init_constant "le")
+let le = function () -> (Coqlib.gen_reference_in_modules "RecursiveDefinition" Coqlib.init_modules "le")
```

Comment:
Why does this one get a special incantation but not the ones around it?

Reply:
Because Funind code is completely broken and uses the wrong accessor functions. It should manipulate constants most of the time, but it uses constrs instead.